Gemfile.lock

```diff
@@ -394,7 +394,7 @@ DEPENDENCIES
   coffee-rails
   dalli
   database_cleaner
-  devise
+  devise (= 3.5.6)
```

Comment:
Ok, let's go the hard version way, no problem. In any case `bundle outdated` would show us whether there is any update pending.